### ISSUE TITLE
Needs a Portfolio

### ISSUE THREAD

**Issue by Rizzpect** [made-by-aniket/codes-#1]
<img src="https://capsule-render.vercel.app/api?type=venom&color=gradient&customColorList=0,2,2,5,30&height=300&section=header&text=ANIKET%20PANDA&fontSize=90&fontColor=fff&animation=fadeIn&fontAlignY=38&desc=🤖%20Gen%20AI%20Explorer%20|%20🧠%20Deep%20Learning%20Enthusiast%20|%20🚀%20Future%20Innovator&descAlignY=55&descAlign=50"/>

<div align="center">

<img src="https://raw.githubusercontent.com/mayhemantt/mayhemantt/Update/svg/Bottom.svg" width="100%"/>

<a href="https://git.io/typing-svg"><img src="https://readme-typing-svg.demolab.com?font=Fira+Code&weight=600&size=28&duration=3000&pause=1000&color=00F7F7&center=true&vCenter=true&multiline=true&repeat=true&width=800&height=120&lines=%E2%9C%A8+Welcome+to+my+Digital+Universe+%E2%9C%A8;I+turn+Data+into+Intelligence;Building+the+Future+with+AI+%F0%9F%94%AE" alt="Typing SVG" /></a>

<br/>

<img src="https://user-images.githubusercontent.com/74038190/212284100-561aa473-3905-4a80-b561-0d28506553ee.gif" width="100%">

</div>

<p align="center">
  <img src="https://komarev.com/ghpvc/?username=aniketpanda&label=Profile%20Views&color=blueviolet&style=for-the-badge" alt="Profile Views"/>
  <img src="https://img.shields.io/badge/Focus-Generative%20AI-brightgreen?style=for-the-badge"/>
  <img src="https://img.shields.io/badge/Lives-India-success?style=for-the-badge"/>
  <img src="https://img.shields.io/badge/Languages-English%20%26%20Hindi-ff69b4?style=for-the-badge"/>
</p>

<br/>

<img align="right" alt="Coding" width="400" src="https://user-images.githubusercontent.com/74038190/229223263-cf2e4b07-2615-4f87-9c38-e37600f8381a.gif">

## 🧬 **About Me**

```python
class AniketPanda:
    def __init__(self):
        self.name = "Aniket Panda"
        self.role = "Gen AI Student"
        self.university = "KL University, Hyderabad"
        self.language_spoken = ["Hindi", "English", "Odia"]
        
    def say_hi(self):
        print("""
        ╔═══════════════════════════════════════════╗
        ║  Thanks for dropping by! I'm passionate   ║
        ║  about pushing the boundaries of what's   ║
        ║  possible with Artificial Intelligence.   ║
        ║                                           ║
        ║  Let's build the future together! 🚀     ║
        ╚═══════════════════════════════════════════╝
        """)

me = AniketPanda()
me.say_hi()
```

<br clear="both"/>

---

<img src="https://user-images.githubusercontent.com/74038190/212284115-f47cd8ff-2ffb-4b04-b5bf-4d1c14c0247f.gif" width="100%">

## 🎓 **Education**

<div align="center">

```
    ╔════════════════════════════════════════════════════════════════════╗
    ║                                                                    ║
    ║   🏛️  K L UNIVERSITY - HYDERABAD CAMPUS                           ║
    ║                                                                    ║
    ║   ┌────────────────────────────────────────────────────────────┐   ║
    ║   │                                                            │   ║
    ║   │   📚 B.Tech in Computer Science                            │   ║
    ║   │   🤖 Specialization: Generative AI                         │   ║
    ║   │   📍 Hyderabad, Telangana, India                           │   ║
    ║   │                                                            │   ║
    ║   │   "Where Innovation Meets Excellence"                      │   ║
    ║   │                                                            │   ║
    ║   └────────────────────────────────────────────────────────────┘   ║
    ║                                                                    ║
    ╚════════════════════════════════════════════════════════════════════╝
```

</div>

---

## 🧠 **AI Interests & Focus Areas**

<div align="center">

<img src="https://readme-typing-svg.demolab.com?font=Orbitron&size=22&duration=2500&pause=400&color=00FFFF&center=true&vCenter=true&width=700&lines=🔮+Large+Language+Models+(LLMs);🎨+Generative+Adversarial+Networks+(GANs);🗣️+Natural+Language+Processing;👁️+Computer+Vision;🧬+Neural+Network+Architectures;🤖+Reinforcement+Learning;📊+Deep+Learning+Frameworks" alt="Typing SVG" />

</div>

<br/>

<table align="center">
<tr>
<td align="center" width="250">

### 🔮 LLMs
```
   ╭─────────────────╮
   │   ┌─────────┐   │
   │   │ GPT-4   │   │
   │   │ LLaMA   │   │
   │   │ Claude  │   │
   │   │ Gemini  │   │
   │   └─────────┘   │
   │  TRANSFORMERS   │
   ╰─────────────────╯
```
**Text Generation**

</td>
<td align="center" width="250">

### 🎨 Diffusion
```
   ╭─────────────────╮
   │   ░░░░░░░░░░░   │
   │   ░░▓▓▓▓▓▓░░░   │
   │   ░░▓████▓░░░   │
   │   ░░▓████▓░░░   │
   │   ░░▓▓▓▓▓▓░░░   │
   │   ░░░░░░░░░░░   │
   │  NOISE → ART    │
   ╰─────────────────╯
```
**Image Synthesis**

</td>
<td align="center" width="250">

### 🗣️ NLP
```
   ╭─────────────────╮
   │    "Hello!"     │
   │        ↓        │
   │   [TOKENIZE]    │
   │        ↓        │
   │   [EMBEDDING]   │
   │        ↓        │
   │    MEANING      │
   ╰─────────────────╯
```
**Language Understanding**

</td>
</tr>
</table>

---

<img src="https://user-images.githubusercontent.com/74038190/212284158-e840e285-664b-44d7-b79b-e264b5e54825.gif" width="100%">

## ⚡ **Tech Arsenal**

<div align="center">

### 🤖 AI/ML Stack
![Python](https://img.shields.io/badge/Python-FFD43B?style=for-the-badge&logo=python&logoColor=blue)
![PyTorch](https://img.shields.io/badge/PyTorch-EE4C2C?style=for-the-badge&logo=pytorch&logoColor=white)
![TensorFlow](https://img.shields.io/badge/TensorFlow-FF6F00?style=for-the-badge&logo=tensorflow&logoColor=white)
![scikit-learn](https://img.shields.io/badge/scikit--learn-F7931E?style=for-the-badge&logo=scikit-learn&logoColor=white)
![Keras](https://img.shields.io/badge/Keras-D00000?style=for-the-badge&logo=keras&logoColor=white)
![OpenCV](https://img.shields.io/badge/OpenCV-27338e?style=for-the-badge&logo=OpenCV&logoColor=white)

### 🔮 Gen AI Tools
![OpenAI](https://img.shields.io/badge/OpenAI-412991?style=for-the-badge&logo=openai&logoColor=white)
![HuggingFace](https://img.shields.io/badge/HuggingFace-FFD21E?style=for-the-badge&logo=huggingface&logoColor=black)
![LangChain](https://img.shields.io/badge/LangChain-1C3C3C?style=for-the-badge&logo=langchain&logoColor=white)
![Streamlit](https://img.shields.io/badge/Streamlit-FF4B4B?style=for-the-badge&logo=streamlit&logoColor=white)
![Gradio](https://img.shields.io/badge/Gradio-F97316?style=for-the-badge&logo=gradio&logoColor=white)

### 💻 Development
![VS Code](https://img.shields.io/badge/VS_Code-007ACC?style=for-the-badge&logo=visual-studio-code&logoColor=white)
![Jupyter](https://img.shields.io/badge/Jupyter-F37626?style=for-the-badge&logo=jupyter&logoColor=white)
![Git](https://img.shields.io/badge/Git-F05032?style=for-the-badge&logo=git&logoColor=white)
![GitHub](https://img.shields.io/badge/GitHub-181717?style=for-the-badge&logo=github&logoColor=white)
![Docker](https://img.shields.io/badge/Docker-2496ED?style=for-the-badge&logo=docker&logoColor=white)
![Linux](https://img.shields.io/badge/Linux-FCC624?style=for-the-badge&logo=linux&logoColor=black)

</div>

---

## 📊 **Skill Matrix**

<div align="center">

```
╔════════════════════════════════════════════════════════════════════════╗
║                        🎯 PROFICIENCY LEVELS                           ║
╠════════════════════════════════════════════════════════════════════════╣
║                                                                        ║
║   Python           ████████████████████████░░░░░░  85%  🐍            ║
║   Machine Learning █████████████████████░░░░░░░░░  75%  🤖            ║
║   Deep Learning    ████████████████████░░░░░░░░░░  70%  🧠            ║
║   Gen AI / LLMs    ██████████████████████████░░░░  90%  🔮            ║
║   NLP              █████████████████████░░░░░░░░░  75%  📝            ║
║   Computer Vision  ██████████████████░░░░░░░░░░░░  65%  👁️            ║
║   Data Analysis    ████████████████████████░░░░░░  80%  📊            ║
║   Git/GitHub       █████████████████████████░░░░░  85%  🔄            ║
║                                                                        ║
╚════════════════════════════════════════════════════════════════════════╝
```

</div>

---

<img src="https://user-images.githubusercontent.com/74038190/212284100-561aa473-3905-4a80-b561-0d28506553ee.gif" width="100%">

## 🚀 **Featured Projects**

<div align="center">

<table>
<tr>
<td width="50%">

### 🤖 AI Chatbot with RAG
```
┌─────────────────────────────────────┐
│  📄 Documents → 🔍 Embeddings      │
│        ↓                            │
│  💬 Query → 🧠 LLM → 📝 Response   │
│                                     │
│  Tech: LangChain, OpenAI, FAISS    │
└─────────────────────────────────────┘
```
**Retrieval-Augmented Generation System**

</td>
<td width="50%">

### 🎨 AI Art Generator
```
┌─────────────────────────────────────┐
│  📝 "A sunset over mountains"       │
│        ↓                            │
│  🔮 Stable Diffusion                │
│        ↓                            │
│  🖼️ [Generated Image]              │
└─────────────────────────────────────┘
```
**Text-to-Image with Diffusion Models**

</td>
</tr>
<tr>
<td width="50%">

### 📊 Sentiment Analyzer
```
┌─────────────────────────────────────┐
│  💬 Text Input                      │
│        ↓                            │
│  🧬 BERT/Transformer                │
│        ↓                            │
│  😊 Positive | 😐 Neutral | 😢 Neg │
└─────────────────────────────────────┘
```
**Deep Learning NLP Pipeline**

</td>
<td width="50%">

### 🔮 Stock Predictor
```
┌─────────────────────────────────────┐
│  📈 Historical Data                 │
│        ↓                            │
│  🧠 LSTM Neural Network             │
│        ↓                            │
│  📊 Future Predictions              │
└─────────────────────────────────────┘
```
**Time Series Forecasting with RNN**

</td>
</tr>
</table>

</div>

---

## 💡 **My AI Philosophy**

<div align="center">

```
    ╔══════════════════════════════════════════════════════════════════════════╗
    ║                                                                          ║
    ║   "Artificial Intelligence is not about replacing human intelligence,    ║
    ║    it's about augmenting it. The best AI solutions are those that       ║
    ║    empower humans to achieve what was previously impossible."            ║
    ║                                                                          ║
    ║                                        — Aniket Panda 🧠                 ║
    ║                                                                          ║
    ╚══════════════════════════════════════════════════════════════════════════╝
```

</div>

---

<img src="https://user-images.githubusercontent.com/74038190/212284115-f47cd8ff-2ffb-4b04-b5bf-4d1c14c0247f.gif" width="100%">

## 📈 **GitHub Analytics**

<div align="center">

<img src="https://github-readme-stats.vercel.app/api?username=aniketpanda&show_icons=true&theme=tokyonight&hide_border=true&bg_color=0D1117&title_color=00FFFF&icon_color=00FFFF&text_color=FFFFFF" width="49%"/>
<img src="https://github-readme-streak-stats.herokuapp.com/?user=aniketpanda&theme=tokyonight&hide_border=true&background=0D1117&stroke=00FFFF&ring=00FFFF&fire=FF6B6B&currStreakLabel=00FFFF" width="49%"/>

<br/><br/>

<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=aniketpanda&layout=compact&theme=tokyonight&hide_border=true&bg_color=0D1117&title_color=00FFFF&text_color=FFFFFF" width="40%"/>

<br/><br/>

<img src="https://github-readme-activity-graph.vercel.app/graph?username=aniketpanda&bg_color=0D1117&color=00FFFF&line=00FFFF&point=FFFFFF&area=true&hide_border=true" width="95%"/>

</div>

---

## 🏆 **Achievements & Goals**

<div align="center">

```
    ┌───────────────────────────────────────────────────────────────────┐
    │                     🎯 MY JOURNEY MAP                             │
    └───────────────────────────────────────────────────────────────────┘
    
         2024                  2025                  2026
           │                     │                     │
           ▼                     ▼                     ▼
    ┌─────────────┐      ┌─────────────┐      ┌─────────────┐
    │ Started     │      │ Building    │      │ Creating    │
    │ Gen AI      │ ───► │ Real-World  │ ───► │ Impactful   │
    │ Journey     │      │ Projects    │      │ AI Systems  │
    └─────────────┘      └─────────────┘      └─────────────┘
           │                     │                     │
           ▼                     ▼                     ▼
    ┌─────────────┐      ┌─────────────┐      ┌─────────────┐
    │ ✅ Python   │      │ ✅ LLMs     │      │ ⏳ Research │
    │ ✅ ML Basics│      │ ✅ RAG Apps │      │ ⏳ Papers   │
    │ ✅ DL Intro │      │ ✅ Fine-tune│      │ ⏳ Startup? │
    └─────────────┘      └─────────────┘      └─────────────┘
```

</div>

---

<img src="https://user-images.githubusercontent.com/74038190/212284158-e840e285-664b-44d7-b79b-e264b5e54825.gif" width="100%">

## 🌐 **Connect With Me**

<div align="center">

<a href="https://linkedin.com/in/aniketpanda"><img src="https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white"/></a>
<a href="https://twitter.com/aniketpanda"><img src="https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white"/></a>
<a href="mailto:aniket.panda@email.com"><img src="https://img.shields.io/badge/Email-D14836?style=for-the-badge&logo=gmail&logoColor=white"/></a>
<a href="https://github.com/aniketpanda"><img src="https://img.shields.io/badge/GitHub-100000?style=for-the-badge&logo=github&logoColor=white"/></a>
<a href="https://kaggle.com/aniketpanda"><img src="https://img.shields.io/badge/Kaggle-20BEFF?style=for-the-badge&logo=kaggle&logoColor=white"/></a>
<a href="https://huggingface.co/aniketpanda"><img src="https://img.shields.io/badge/HuggingFace-FFD21E?style=for-the-badge&logo=huggingface&logoColor=black"/></a>

<br/><br/>

```
╭─────────────────────────────────────────────────────────────────────────╮
│                                                                         │
│   📧 Email:    aniket.panda@email.com                                   │
│   🏫 College:  KL University, Hyderabad                                 │
│   📍 Location: Hyderabad, India                                         │
│   🎯 Focus:    Generative AI & Deep Learning                            │
│                                                                         │
│   ┌─────────────────────────────────────────────────────────────────┐   │
│   │  "Open to collaborations, internships, and AI research projects │   │
│   │   Let's build something amazing together! 🚀"                    │   │
│   └─────────────────────────────────────────────────────────────────┘   │
│                                                                         │
╰─────────────────────────────────────────────────────────────────────────╯
```

</div>

---

## 🎵 **Currently Vibing To**

<div align="center">

[![Spotify](https://novatorem-spotify-tau.vercel.app/api/spotify)](https://open.spotify.com/user/aniketpanda)

</div>

---

## 🐍 **Contribution Snake**

<div align="center">

![Snake animation](https://github.com/aniketpanda/aniketpanda/blob/output/github-contribution-grid-snake-dark.svg)

</div>

---

<div align="center">

### 💭 Random Dev Quote
![](https://quotes-github-readme.vercel.app/api?type=horizontal&theme=tokyonight)

</div>

---

<div align="center">

## 🔥 **Fun Fact**

```
    ╔══════════════════════════════════════════════════════════════════╗
    ║                                                                  ║
    ║   🧠 Did you know?                                               ║
    ║                                                                  ║
    ║   GPT-4 has approximately 1.76 TRILLION parameters!             ║
    ║   That's more than the number of stars in the Milky Way! 🌌     ║
    ║                                                                  ║
    ║   And I'm here trying to understand how it all works... 😅      ║
    ║                                                                  ║
    ╚══════════════════════════════════════════════════════════════════╝
```

</div>

---

<img src="https://capsule-render.vercel.app/api?type=shark&color=gradient&customColorList=0,2,2,5,30&height=150&section=footer&text=Let's%20Build%20the%20Future%20with%20AI%20🚀&fontSize=24&fontColor=fff&animation=twinkling"/>

<div align="center">

<img src="https://readme-typing-svg.demolab.com?font=Orbitron&size=25&duration=4000&pause=1000&color=00FFFF&center=true&vCenter=true&width=600&lines=Thanks+for+visiting!+%F0%9F%99%8F;Let's+connect+and+collaborate!;Building+AI+for+a+better+tomorrow+%E2%9C%A8" alt="Typing SVG" />

<br/>

![Visitor Count](https://profile-counter.glitch.me/aniket-panda-portfolio/count.svg)

<br/>

**⭐ If you like my work, consider giving a star! ⭐**

<img src="https://raw.githubusercontent.com/Trilokia/Trilokia/379277808c61ef204768a61bbc5d25bc7798ccf1/bottom_header.svg" />

</div>

**Comment by Rizzpect** [comment]
Thankyou for resolving the issue.
